lang/de/block_navbuttons.php

```diff
@@ -33,4 +33,7 @@
 $string['selftest_nofooter'] = 'Navigationstaste Selbsttest: Die Zeile \"include(\$CFG->dirroot.\'/blocks/navbuttons/footer.php\');\" wurde nicht korrekt weblib.php zugeordnet - siehe README.txt f&uuml;r weitere Anweisungen';
```

Comment:
Please keep all the language files in alphabetical order - it makes them much easier to maintain.